tests/lib/rules/prefer-destructuring.js

```diff
@@ -83,15 +93,23 @@ ruleTester.run("prefer-destructuring", rule, {
             }]
         },
         {
-            code: "var foo = array.foo;",
+            code: "var foo = object.foo;",
+            errors: [{
+                message: "Use object destructuring",
+                type: "VariableDeclarator"
+            }]
+        },
+        {
+            code: "var foobar = object.bar;",
+            options: [{ object: true }, { enforceForRenamedProperties: true }],
             errors: [{
                 message: "Use object destructuring",
                 type: "VariableDeclarator"
             }]
         },
         {
-            code: "var foo = array.bar;",
-            options: [{ object: true }, { requireRenaming: true }],
+            code: "var foo = object[bar];",
```

Comment:
i'm not sure i understand - `object[bar]` isn't the same as `object.bar`. How would you use destructuring here? Does `var { [bar]: foo } = object;` work?